src/data/options.ts

```diff
@@ -1 +1,7 @@
 export const paginationOptions = [10, 20, 30, 40, 50]
+export const roleOptions = [
+  { label: 'Admin', value: 'admin' },
+  { label: 'User', value: 'user' },
```

Comment:
this seems to be duplicate again. as said previously keep one source of truth for this

export const roleOptions = [
  { label: 'Admin', value: 'admin' },
  { label: 'User', value: 'user' },
} as const
export const roleTuple = roleOptions.map(r=>r.value) // this will be auto typed anyway due to as const

export type Role = typeof roleOptions[0]['value']

export const roleSchema = z.enum(roleTuple)

Reply:
updated!